# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: target/mnemosyne/pkg_patch/mozplugger.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

disable conflicts with mplayerplug-in
added support for ooffice2.0 and xfmedia

--- ./mozpluggerrc.orig	2005-11-16 16:43:28.000000000 -0300
+++ ./mozpluggerrc	2005-11-16 17:13:38.000000000 -0300
@@ -70,6 +70,7 @@
 
 define([OO],[swallow(OpenOffice) fill: openoffice -nologo $1 "$file"
 	swallow(OpenOffice) fill: ooffice -nologo $1 "$file"
+	swallow(OpenOffice) fill: ooffice2.0 -nologo $1 "$file"
 	swallow(OpenOffice) fill: soffice -nologo $1 "$file"])
 
 ### Acrobat Reader
@@ -89,65 +90,6 @@
 define(GV_FLAGS,[repeat noisy swallow(gv) fill])
 define(GV,[GV_FLAGS(): gv GV_OPTS() "$file"])
 
-###################
-### Video       ###
-###################
-
-video/mpeg:mpeg,mpg,mpe:MPEG animation
-video/x-mpeg:mpeg,mpg,mpe:MPEG animation
-video/x-mpeg2:mpv2,mp2ve:MPEG2 animation
-	MP_VIDEO_STREAM()
-	TM_VIDEO_STREAM()
-        HELPER(xine -pq "$file")
-	loop: mtvp -l -W$window "$file"
-	: mtvp -W$window "$file"
-	loop: xanim +Av100 -Zr +W$window +q +f "$file"
-	: xanim +Av100 -Zr +W$window +q +Ze +f "$file"
-
-video/mp4:mp4:MPEG4 animation
-	MP_VIDEO_STREAM()
-	TM_VIDEO_STREAM()
-	HELPER(xine -pq "$file")
-
-video/msvideo:avi:AVI animation
-video/x-msvideo:avi:AVI animation
-video/fli:fli,flc:FLI animation
-video/x-fli:fli,flc:FLI animation
-	MP_VIDEO_STREAM()
-	TM_VIDEO_STREAM()
-	HELPER(xine -pq "$file")
-
-application/x-mplayer2:wmv,asf,mov:Windows Media
-video/x-ms-asf:asf,asx,wma,wax,wmv,wvx:Windows Media
-video/x-ms-wmv:wmv:Windows Media
-	MP_VIDEO_STREAM()
-	TM_VIDEO_STREAM()
-
-application/x-quicktimeplayer:mov:Quicktime animation
-image/x-macpaint:pntg,mov:Quicktime animation
-video/quicktime:mov,qt:Quicktime animation
-video/x-quicktime:mov,qt:Quicktime animation
-	MP_VIDEO_STREAM()
-	TM_VIDEO_STREAM()
-	HELPER(xine -pq "$file")
-
-video/x-theora:ogg:OGG stream with video
-video/theora:ogg:OGG stream with video
-video/ogg:ogg:OGG stream with video
-video/x-ogg:ogm,ogv:OGG stream with video
-ifdef([MP_VORBIS],[	MP_VIDEO_STREAM()
-])	TM_VIDEO_STREAM()
-
-video/dl:dl:DL animation
-video/x-dl:dl:DL animation
-video/sgi-movie:movie,movi,mv:SGI animation
-video/x-sgi-movie:movie,movi,mv:SGI animation
-video/anim:iff,anim5,anim3,anim7:IFF animation
-video/x-anim:iff,anim5,anim3,anim7:IFF animation
-	loop: xanim +Av100 -Zr +W$window +q +f "$file"
-	: xanim +Av100 -Zr +W$window +q +Ze +f "$file"
-
-
 ##################
 ### Audio      ###
 ##################
@@ -166,25 +108,6 @@
 	controls loop noisy: xmp -l --nocmd "$file"
 	controls noisy: xmp --nocmd "$file"
 
-audio/mp3:mp3:MPEG audio
-audio/x-mp3:mp3:MPEG audio
-audio/mpeg2:mp2:MPEG audio
-audio/x-mpeg2:mp2:MPEG audio
-audio/mpeg3:mp3:MPEG audio
-audio/x-mpeg3:mp3:MPEG audio
-audio/mpeg:mpa,abs,mpega:MPEG audio
-audio/x-mpeg:mpa,abs,mpega:MPEG audio
-	MP_AUDIO_STREAM()
-	TM_AUDIO_STREAM()
-	controls: mpg321 -q "$file"
-	controls: mpg123 -q "$file"
-	controls: splay -t 200 "$file"
-	controls: amp -b 200 -q "$file"
-	controls: maplay "$file"
-	controls: mpeg3play "$file"
-	HELPER(xmms -e -p "$file")
-	repeat noisy swallow(alsaplayer): alsaplayer -q "$file"
-
 audio/mpeg-url:m3u:MPEG music resource locator
 audio/x-mpeg-url:m3u:MPEG music resource locator
 audio/mpegurl:m3u:MPEG music resource locator
@@ -193,14 +116,7 @@
 audio/x-mpeg-url:m3u:MPEG music resource locator
 audio/x-scpls:pls:Shoutcast Playlists
 #	controls: mpg321 -q -@ "$file"
-	HELPER(xmms -e -p "$file")
-
-audio/x-ogg:ogg:OGG audio
-application/x-ogg:ogg:OGG audio
-application/ogg:ogg:OGG audio
-ifdef([MP_VORBIS],[	MP_AUDIO_STREAM()
-])	TM_AUDIO_STREAM()
-	controls stream noisy: ogg123 -q -b 128 "$file"
+	HELPER(xfmedia -p "$file")
 	HELPER(xmms -e -p "$file")
 
 audio/x-sidtune:sid,psid:Commodore 64 Audio
@@ -214,31 +130,6 @@
 	controls: play "$file"
 	controls: sox "$file" -t .au - > /dev/audio
 
-audio/wav:wav:Microsoft wave file
-audio/x-wav:wav:Microsoft wave file
-audio/x-pn-wav:wav:Microsoft wave file
-audio/x-pn-windows-acm:wav:Microsoft wave file
-        controls: play "$file"
-	controls: wavplay -q "$file"
-	controls noisy: bplay "$file"
-	controls: splay "$file"
-	HELPER(xmms -e -p "$file")
- 	repeat noisy swallow(alsaplayer): alsaplayer -q "$file"
-
-audio/x-pn-realaudio-plugin:rpm:RealPlayer Plugin Metafile
-audio/x-pn-realaudio:ra,rm,ram:Realaudio-plugin resource locator
-audio/x-realaudio:ra,rm,ram:RealAudio file
-application/vnd.rn-realmedia:rm:RealMedia file
-application/smil:smi:RealPlayer
-audio/vnd.rn-realaudio:ra,ram:RealAudio file
-audio/vnd.rn-realvideo:rv:RealVideo file
-	nokill stream: hxplay "$file"
-        nokill stream: realplay "$file"
-
-audio/x-ms-wax:wax,wma:Windows Media
-	MP_AUDIO_STREAM()
-	TM_AUDIO_STREAM()
-
 #######################
 ### Documents       ###
 #######################
